libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4980,6 +4981,34 @@ MAV_RESULT GCS_MAVLINK::handle_command_int_do_set_home(const mavlink_command_int
     return MAV_RESULT_ACCEPTED;
 }
 
+MAV_RESULT GCS_MAVLINK::handle_command_int_external_position_estimate(const mavlink_command_int_t &packet)
+{
+
+    if (!isnan(packet.z)) {
+        // ensure sender doesn't want altitude, which we don't support
+        return MAV_RESULT_UNSUPPORTED;
```

Comment:
```suggestion
        return MAV_RESULT_DENIED;
```
It is supported, it just has invalid params.

Reply:
done